bibliography.bib

```diff
@@ -7142,3 +7142,474 @@ @Article {ProHsiKum2017IEEETR
+    booktitle    = icra_2022,
+    editor       = {},
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {2022},
+    pages        = {2891--2897},
+    publisher    = pub_IEEE,
+    address      = {},
+    doi          = {10.1109/ICRA46639.2022.9812446},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings {ZhoSma2021aaai,
+    title        = {Inverse reinforcement learning with natural language goals},
+    author       = Zhou_L # and # Small_K,
+    booktitle    = aaai_2021,
+    editor       = {},
+    series       = {},
+    volume       = {35},
+    number       = {12},
+    year         = {2021},
+    pages        = {11116--11124},
+    publisher    = pub_IEEE,
+    address      = {},
+    doi          = {10.1609/aaai.v35i12.17326},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings{HanAndMadBogh2020sii,
+    title        = {Transferring human manipulation knowledge to robots with inverse reinforcement learning},
+    author       = Hansen_EB # and # Andersen_RE # and # Madsen_S # and # Bogh_S,
+    booktitle    = sii_2020,
+    editor       = {},
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {2020},
+    pages        = {933--937},
+    publisher    = pub_IEEE,
+    address      = {},
+    doi          = {10.1109/SII46433.2020.9025873},
+    note         = {},
+    annote       = {},
+}
+
+@Article{OsaPajNeu-etal2018FTR,
+    title        = {An algorithmic perspective on imitation learning},
+    author       = Osa_T # and # Pajarinen_J # and # Neumann_G # and # Bagnell_JA # and # Abbeel_P # and # Peters_J,
+    journal      = FTR,
+    series       = {},
+    volume       = {7},
+    number       = {1--2},
+    year         = {2018},
+    pages        = {1--179},
+    doi          = {10.1561/2300000053},
+    note         = {},
+    annote       = {},
+}
+
+@InProceedings{NatKunJud-etal2010icmla,
+    title        = {Multi-agent inverse reinforcement learning},
+    author       = Natarajan_S # and # Kunapuli_G # and # Judah_K # and # Tadepalli_P # and # Kersting_K # and # Shavlik_J,
+    booktitle    = icmla_2010,
+    editor       = {},
+    series       = {},
+    volume       = {},
+    number       = {},
+    year         = {2010},
+    pages        = {395--400},
+    publisher    = pub_IEEE,
+    address      = {},
+    doi          = {10.1109/ICMLA.2010.65},
+    note         = {},
+    annote       = {},
+}
+
+@Article{Mah1996ML,
+    title        = {Average reward reinforcement learning: foundations, algorithms, and empirical results},
+    author       = Mahadevan_S,
+    journal      = ML,
+    series       = {},
+    volume       = {22},
+    number       = {1},
+    year         = {1996},
+    pages        = {159--195},
+    doi          = {10.1023/A:1018064306595},
+    note         = {},
+    annote       = {},
+}
+
+@Article{SonRenSadErm2018ANIPS,
```

Comment:
This should be an InProceedings, not Article (see comment in journal.bib). Also, stick to the template (no url) and please try to find at least the doi, but preferably also some of the missing info (series, pages, number...)